packages/teams-js/test/utils.ts

```diff
@@ -159,6 +168,23 @@ export class Utils {
     } as MessageEvent);
   };
 
+  public respondToOpenerMessage = (message: MessageRequest, ...args: unknown[]): void => {
```

Comment:
>respondToOpenerMessage [](http://example.com/codeflow?start=9&length=22)

I wasn't sure I could guess what this function did based on the name. After looking at the implementation, should it be something like "respondToMessageAsOpener" ? The opener part is only affecting the `source` property, is that right? The current name makes it sound like there's a thing called "the opener message" and this is about responding to that.